### PR TITLE
fix: Set default theme when application is exported

### LINKED ISSUE #13137: NullPointerException
Sentry Issue: [APPSMITH-2A9](https://sentry.io/organizations/appsmith/issues/3189403998/?referrer=github_integration)

```
NullPointerException: None
    at com.appsmith.server.helpers.GitFileUtils.saveApplicationToLocalRepo(GitFileUtils.java:99)
```

### PR DESCRIPTION
**Pull request by nayan-rafiq**
## Description
When application is exported and it does not have a theme set, a NullPointerException was raised in that case. This PR fixes that by returning the default theme when theme is missing.

Fixes #13137

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- JUnit tests
- Manually

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/server/appsmith-server/src/test/java/com/appsmith/server/solutions/ImportExportApplicationServiceTests.java [hunk 1]
```diff
@@ -2397,5 +2398,31 @@ public void exportAndImportApplication_withMultiplePages_PagesOrderIsMaintained(
...
+        customTheme.setName("my-custom-theme");
+
+        String randomId = UUID.randomUUID().toString();
+        Application testApplication = new Application();
+        testApplication.setName("Application_" + randomId);
+        Mono<ApplicationJson> exportedAppJson = applicationPageService.createApplication(testApplication, orgId)
+                //.zipWith(themeRepository.save(customTheme))
+                .flatMap(application -> {
+                    //Application application = objects.getT1();
+                    // Theme theme = objects.getT2();
```
Comment (abhvsn) [inline app/server/appsmith-server/src/test/java/com/appsmith/server/solutions/ImportExportApplicationServiceTests.java:2415]: Please remove the comments

**Review by abhvsn — COMMENTED** [review]
LGTM minor comment

File: app/server/appsmith-server/src/test/java/com/appsmith/server/solutions/ImportExportApplicationServiceTests.java (hunk 1 above)
Comment (nayan-rafiq) [reply app/server/appsmith-server/src/test/java/com/appsmith/server/solutions/ImportExportApplicationServiceTests.java]: Done

**Review by abhvsn — APPROVED** [review]

**Pull request merged**